index.js

```diff
@@ -1,18 +1,51 @@
 #!/usr/bin/env node
+var assert = require('assert');
 require('shelljs/global');
 
 // Creates a changelog for the current build and puts it in the root
 // Commits the changelog if it updated
 // Does not push commit
 
+// The maximum number of deletions for updating the changelog should
+// be 2 lines. Typically, the "Unreleased" and "Full Changelog" links
+// are changed because the version increases.
+var MAX_DELETIONS = 2;
```

Comment:
Let's increase this a little bit in case someone changes the name of a pull request or Github issue. Maybe 4 or 5 would be ok?